privacyidea/static/components/token/views/token.html

```diff
@@ -80,9 +80,32 @@
                         <a ui-sref="token.getserial">
                             <span class="glyphicon glyphicon-question-sign">
                             </span>
-                            <span translate>Get Serial</span>
-                        </a>
-                    </li>
+                            <span translate>Get Serial</span></a></li>
+                    <hr>
+                    <li role="presentation"
+                        ng-show="($state.includes('token.list'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/webui/index.html#tokens'}}">
+                            <span class="glyphicon glyphicon-info-sign"></span>
+                            <span translate>Help about Token</span></a></li>
+                    <li role="presentation"
+                        ng-show="($state.includes('token.enroll'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/tokens/tokentypes.html#'}}">
+                            <span class="glyphicon glyphicon-info-sign"></span>
+                            <span translate>Help about Tokentypes</span></a></li>
+                     <li role="presentation"
+                        ng-show="($state.includes('token.import'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/workflows_and_tools/import/index.html#'}}">
+                            <span class="glyphicon glyphicon-info-sign"></span>
+                            <span translate>Help about Import</span></a></li>
```

Comment:
Maybe `Token Import` would be more accurate.

Reply:
done in a37f2d98d2d6876d137ba6a6783b086b289941c3.